tool/tctl/common/plugins_command.go

```diff
@@ -45,10 +80,68 @@ func (p *PluginsCommand) Initialize(app *kingpin.Application, config *servicecfg
 	p.dryRun = true
 
 	pluginsCommand := app.Command("plugins", "Manage Teleport plugins.").Hidden()
+
 	p.cleanupCmd = pluginsCommand.Command("cleanup", "Cleans up the given plugin type.")
 	p.cleanupCmd.Arg("type", "The type of plugin to cleanup. Only supports okta at present.").Required().EnumVar(&p.pluginType, string(types.PluginTypeOkta))
 	p.cleanupCmd.Flag("dry-run", "Dry run the cleanup command. Dry run defaults to on.").Default("true").BoolVar(&p.dryRun)
 
+	p.initInstall(pluginsCommand)
+	p.initDelete(pluginsCommand)
+}
+
+func (p *PluginsCommand) initInstall(parent *kingpin.CmdClause) {
+	p.install.cmd = parent.Command("install", "Install new plugin instance")
+
+	p.install.scim.cmd = p.install.cmd.Command("scim", "Install a new SCIM integration")
+	p.install.scim.cmd.
+		Flag("name", "The name of the SCIM plugin resource to create").
+		Default("scim").
+		StringVar(&p.install.name)
+	p.install.scim.cmd.
+		Flag("saml-connector", "The name of the Teleport SAML connector users will log in with.").
+		Required().
+		StringVar(&p.install.scim.samlConnector)
+	p.install.scim.cmd.
+		Flag("role", "The Teleport role to assign users created by the plugin").
+		Short('r').
+		Default(teleport.PresetRequesterRoleName).
+		StringVar(&p.install.scim.role)
+	p.install.scim.cmd.
+		Flag("token", "The bearer token used by the SCIM client to authenticate").
+		Short('t').
+		Required().
+		Envar("TELEPORT_SCIM_BEARER_TOKEN").
+		StringVar(&p.install.scim.token)
+	p.install.scim.cmd.
+		Flag("force", "Proceed with installation even if validation fails").
+		Short('f').
+		Default("false").
+		BoolVar(&p.install.scim.force)
+
+}
+
+func (p *PluginsCommand) initDelete(parent *kingpin.CmdClause) {
+	p.delete.cmd = parent.Command("delete", "Remove a plugin instance")
+	p.delete.cmd.
+		Arg("name", "The name of the SCIM plugin resource to delete").
```

Comment:
is the name supposed to be specific to a SCIM plugin? we say it generically else where like `Remove a plugin instance`

Reply:
Yeah, this command would allow multiple instances of the same Plugin type, disambiguated by name